sphinxcontrib/drawio/deprecated.py

```diff
@@ -166,7 +166,9 @@ def render_drawio(
     elif platform.system() == "Windows":
         binary_path = r"C:\Program Files\draw.io\draw.io.exe"
     else:
-        binary_path = "/opt/draw.io/drawio"
```

Comment:
Please don't update the deprecated file. It should remain the same.

Reply:
This means we'll have to change a bunch of the test cases and remove some in total (see the last commit)